channels/unsorted.m3u

```diff
@@ -1 +1,379 @@
+http://m6gdaqgeln93-hls-live.5centscdn.com/metro/d0dbe915091d400bd8ee7f27f0791303.sdp/index.m3u8
+#EXTINF:-1 tvg-logo="http://mhdtvworld.com/wp-content/uploads/2019/03/JAFFANTV.png" group-title="Local",Jaffna TV
+http://59d39900ebfb8.streamlock.net:1935/Jaffnatv/Jaffnatv/playlist.m3u8
+#EXTINF:-1 tvg-logo="http://mhdtvworld.com/wp-content/uploads/2019/03/astro_vaanavil.png" group-title="Entertainment",Astro Vaanavil
+http://playbackn18.aotg-video.astro.com.my/CH1/master_VAAN_8_05.m3u8
+#EXTINF:-1 tvg-logo="http://mhdtvworld.com/wp-content/uploads/2019/03/Screenshot_1-1.png" group-title="Entertainment",ETV Telugu
+http://195.181.169.151:8080/ETVTelugu-telu/index.m3u8
+#EXTINF:-1 tvg-logo="http://mhdtvworld.com/wp-content/uploads/2019/08/ETVUS.png" group-title="Entertainment",ETV Telugu US
+https://msluseasts.akamaized.net/hls/live/2000128/etvusmsl/playlist.m3u8
+#EXTINF:-1 tvg-logo="http://mhdtvworld.com/wp-content/uploads/2019/03/RAGAMTV.jpg" group-title="Local",Ragam TV
+http://account11.livebox.co.in/ntvhls/ntv.m3u8
+#EXTINF:-1 tvg-logo="http://mhdtvworld.com/wp-content/uploads/2019/03/Screenshot_1.png" group-title="Local",Tamil MTV
+http://santhora.de.flash.glb.ipercast.net/santhora.de-live/tamilmtv/tamilmtv/playlist.m3u8
+#EXTINF:-1 tvg-logo="http://mhdtvworld.com/wp-content/uploads/2019/03/watamantv.png" group-title="Local",Wathanam TV
+http://95.216.167.183:5080/LiveApp/streams/443106610169904881506470.m3u8
+#EXTINF:-1 tvg-logo="http://mhdtvworld.com/wp-content/uploads/2019/03/CTAMIL.png" group-title="Local",C Tamil TV
+http://ctamiltv.livebox.co.in/ctamiltvhls/live.m3u8
+#EXTINF:-1 tvg-logo="http://mhdtvworld.com/wp-content/uploads/2019/02/yuppclassicsmarathi.png" group-title="Movies",Classics Marathi
+https://d1fi19tywmn14b.cloudfront.net/yuppindmov/ngrp:ymarcla.stream_all/playlist.m3u8
+#EXTINF:-1 tvg-logo="http://mhdtvworld.com/wp-content/uploads/2018/12/Sony_TV.png" group-title="Entertainment",Sony TV
+http://130.185.150.79:1935/trunk/SONY-HD/playlist.m3u8
+#EXTINF:-1 tvg-logo="http://mhdtvworld.com/wp-content/uploads/2019/01/unnamed.jpg" group-title="Entertainment",Vasantham
+https://d39v9xz8f7n8tk.cloudfront.net//hls//vsnthmctv//master.m3u8
+#EXTINF:-1 tvg-logo="http://mhdtvworld.com/wp-content/uploads/2019/01/b4u_plus_ae.png" group-title="Movies",B4U Plus
+http://livecdnh3.tvanywhere.ae/hls/b4uplus/04.m3u8
+#EXTINF:-1 tvg-logo="http://mhdtvworld.com/wp-content/uploads/2019/01/zee-aflam-in.png" group-title="Movies",Zee Aflam
+http://livecdnh3.tvanywhere.ae/hls/zeeaflam/01.m3u8
+#EXTINF:-1 tvg-logo="http://mhdtvworld.com/wp-content/uploads/2019/01/Screenshot_1.png" group-title="Movies",Sony Max HD
+https://msleurope2s.akamaized.net/hls/live/2000183/setmaxukhdmsl/playlist.m3u8
+#EXTINF:-1 tvg-logo="http://mhdtvworld.com/wp-content/uploads/2018/12/Sony_Pix_HD.png" group-title="",Sony Pix HD
+http://xtreamsports.xyz/jio/jiotvxs.php?c=Sony_Pix_HD&q=600&e=.m3u8
```

Comment:
http://xtreamsports.xyz/jio/jiotvxxxxXX.php?c=Sony_Pix_HD&q=600&e=.m3u8

Reply:
removed

Reply:
look also to new file added there many india link from http://mhdtvsports.com are no working. i find many of them

Reply:
there are no such links in the current version of playlists